src/mcp/server/lowlevel/server.py

```diff
@@ -83,7 +83,7 @@ async def main():
 from mcp.server.lowlevel.helper_types import ReadResourceContents
 from mcp.server.models import InitializationOptions
 from mcp.server.session import ServerSession
-from mcp.server.stdio import stdio_server as stdio_server
+from mcp.server.stdio import stdio_server as stdio_server  # noqa: PLC0414
```

Comment:
Why is `# noqa: PLC0414` necessary? Would prefer to avoid adding lint-ignores.

```
PLC0414 | useless-import-alias | Import alias does not rename original package

PLC0414	[useless-import-alias](https://docs.astral.sh/ruff/rules/useless-import-alias/)	Import alias does not rename original package
```

Could we instead just change this to:

```
from mcp.server.stdio import stdio_server
```